ticdc/ticdc-sink-to-kafka.md

```diff
@@ -236,7 +236,7 @@ You can use `partition = "xxx"` to specify a partition dispatcher. It supports f
 
 ## Scale out the load of a single large table to multiple TiCDC nodes
 
-This feature splits a single large table into multiple data ranges based on the number of Regions, and distributes these data ranges to multiple TiCDC nodes so that multiple TiCDC nodes can replicate the large single table at the same time. This feature can solve the following two problems:
+This feature splits the data replication range of a single large table into multiple ranges, according to the number of Regions and the number of modified rows per minute, and it makes the number of Regions and the number of modified rows replicated between these ranges approximately the same. This feature distributes these ranges to multiple TiCDC nodes for replication, so that multiple TiCDC nodes can replicate a large single table at the same time. This feature can solve the following two problems:
```

Comment:
```suggestion
This feature splits the data replication range of a single large table into multiple ranges, according to the data volume and the number of modified rows per minute, and it makes the data volume and the number of modified rows replicated in each range approximately the same. This feature distributes these ranges to multiple TiCDC nodes for replication, so that multiple TiCDC nodes can replicate a large single table at the same time. This feature can solve the following two problems:
```